html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Html forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
+      <div>
+        <input type="text" id="email" name="email" />
+      </div>
+      <div>
+        <label for="coding">Why is coding cool?</label>
+      </div>
+      <div>
+        <select id="coding" name="coding">
+          <option>Select an option</option>
+          <option>Because it's interesting</option>
+          <option>Because it's fun</option>
+          <option>Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label for="language">Favorite Language</label>
```

Comment:
This doesn't correlate to a specific form control element, so it doesn't need to be a label. You can make this a general text element such as a `p` element.